qemu/tests/hotplug_mem_negative.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import re
+
+from avocado.utils import memory
+
+from virttest import env_process
+from virttest import error_context
+
+from virttest.qemu_devices import qdevices
+from virttest.utils_numeric import normalize_data_size
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu memory hotplug test:
+    1) Boot guest with -m option.
+    2) Hotplug memory with invalid params.
+    3) Check qemu prompt message.
+    4) Check vm is alive after hotplug.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    @error_context.context_aware
+    def _hotplug_memory(vm, name):
+        hotplug_test = MemoryHotplugTest(test, params, env)
+        devices = vm.devices.memory_define_by_params(params, name)
+        for dev in devices:
+            if isinstance(dev, qdevices.Dimm):
+                if params["set_addr"] == "yes":
+                    addr = params["addr_dimm_%s" % name]
+                else:
+                    addr = hotplug_test.get_mem_addr(vm, dev.get_qid())
+                dev.set_param("addr", addr)
+            error_context.context("Hotplug %s '%s' to VM" %
+                                  ("pc-dimm", dev.get_qid()), logging.info)
+            vm.devices.simple_hotplug(dev, vm.monitor)
```

Comment:
@zhenyzha @huangyum Do we need to check the `ver_out` status here?

Reply:
I think don't need to check the ver_out status, 
because this is a negative test, the status output of success or failure will be collected in def collect_hotplug_info ()